build.sbt

```diff
@@ -162,7 +162,10 @@ lazy val userImpl = (project in file("user-impl"))
   .dependsOn(userApi)
   .settings(
     version := "1.0-SNAPSHOT",
-    libraryDependencies += lagomJavadslPersistenceCassandra
+    libraryDependencies ++= Seq(
+      lagomJavadslPersistenceCassandra,
+      "org.mindrot" % "jbcrypt" % "0.3m"
```

Comment:
It looks like there is a more recent version 0.4. It is probably a good idea to use that instead.s